app/services/to_cocina/collection_mapper.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module ToCocina
+  # Maps CollectionForm to Cocina Collection
+  class CollectionMapper
+    def self.call(...)
+      new(...).call
+    end
+
+    # @param [CollectionForm] collection_form
+    # @param [source_id] source_id
+    def initialize(collection_form:, source_id:)
+      @collection_form = collection_form
+      @content = content
+      @source_id = source_id
+    end
+
+    # @return [Cocina::Models::Collection]
+    def call
+      if collection_form.persisted?
+        Cocina::Models.with_metadata(Cocina::Models.build(params), collection_form.lock)
+      else
+        Cocina::Models.build_request(params)
+      end
+    end
+
+    private
+
+    attr_reader :collection_form, :source_id, :content
+
+    def params
+      {
+        externalIdentifier: collection_form.druid,
+        type: Cocina::Models::ObjectType.collection,
+        label: collection_form.title,
+        description: ToCocina::DescriptionMapper.call(form: collection_form),
```

Comment:
Is using the DescriptionMapper going to be problematic when we add additional fields that collections don't have?

Reply:
It shouldn't as collections allow the same descriptive spec as DROs do. But if we'd prefer this to be separate to avoid confusion that's an easy change.